app/user/rest.py

```diff
@@ -179,17 +182,20 @@ def verify_user_code(user_id):
     validate(data, post_verify_code_schema)
 
     user_to_verify = get_user_by_id(user_id=user_id)
+    check_throttle_for_requester(user_to_verify)
```

Comment:
It looks like we've already checked the password and returned an error before this point, we specifically don't want to allow someone to probe for a password that is correct, perhaps move this check to the first thing in the function?